src/plugins/azgenerator/Generator.ts

```diff
@@ -32,16 +32,16 @@ export async function GenerateAll(model: CodeModelAz,
         {
             let pathTop = "src/" + model.Extension_Name + "/";
             let path = "src/" + model.Extension_Name + "/azext_" + model.Extension_Name.replace("-", "_") + "/";
-            files[path + "_help.py"] = GenerateAzureCliHelp(model);
-            files[path + "_params.py"] = GenerateAzureCliParams(model);
-            files[path + "commands.py"] = GenerateAzureCliCommands(model);
-            files[path + "custom.py"] = GenerateAzureCliCustom(model);
-            files[path + "_client_factory.py"] = GenerateAzureCliClientFactory(model);
+            files[path + "generated/_help.py"] = GenerateAzureCliHelp(model);
+            files[path + "generated/_params.py"] = GenerateAzureCliParams(model);
+            files[path + "generated/commands.py"] = GenerateAzureCliCommands(model);
+            files[path + "generated/custom.py"] = GenerateAzureCliCustom(model);
+            files[path + "generated/_client_factory.py"] = GenerateAzureCliClientFactory(model);
```

Comment:
Do we need add a __init__.py file under 'generated' folder?

Reply:
no, just __init__.py in the root